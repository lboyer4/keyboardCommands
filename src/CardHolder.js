import React, { Component } from 'react';
import './CardHolder.scss';