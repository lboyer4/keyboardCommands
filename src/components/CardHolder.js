import React from 'react';
import '../SCSS/_CardHolder.scss';
import Card from './Card.js';
import Rainbow from '../images/rainbow.png';

function CardHolder({ level, currentCard, setCurrentCard, setIncorrectStack, updateScore, setWrongCards, incorrectCards}) { 
	return (
		<section className="card-holder">
			<img className="rainbow" src={Rainbow} alt="magical rainbow" />
			<div className="cards">
				<h4> Level: {level} </h4>
				<h6 className="instructions">
					To Play, enter a command in the input field, and then click "submit answer"
				</h6>
				<p className="example"> Example answer: "command Z"
				</p>
				<Card 
					updateScore={updateScore}
					setIncorrectStack={setIncorrectStack}
					setWrongCards={setWrongCards}
					setCurrentCard={setCurrentCard}
					task={currentCard.task}
					command={currentCard.command}
					level={currentCard.level}
					incorrectCards={incorrectCards}
				/>
			</div>
		</section>
	)
}

export default CardHolder;