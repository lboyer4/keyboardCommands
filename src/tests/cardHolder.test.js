import React from 'react';
import ReactDOM from 'react-dom';
import CardHolder from '../components/CardHolder';
import { shallow } from 'enzyme'; 


describe("CardHolder", () => {
	let wrapper;

	beforeEach(() => {
		wrapper = shallow(
			<CardHolder/>
		)
	});

	it('should match snap shot', () => {
		expect(wrapper).toMatchSnapshot();
	});

	
});