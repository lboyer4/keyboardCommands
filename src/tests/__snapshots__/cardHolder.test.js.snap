// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardHolder should match snap shot 1`] = `undefined`;
